### ISSUE TITLE
Selecting custom Polygons

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#197]
## User story
1. As a User
2. I want to select a custom polygon region
3. So that I get the data of a specific region

## Acceptance criteria
* Create polygons of different shapes (circles, squares, etc.)
* Visualize these polygons in a specific way
* Fetch data about these selected regions (polygons)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Duplicate to #195 close
